### PR TITLE
Create installation and quick start guide pages

### PR DESCRIPTION
**Pull request by alexyorke**
This creates the installation and quick start guide pages (ref task 1887) with the content for those pages.

The following URLs now point to those two pages respectively (and were already in the sidebar):

- `docs/installation/`
- `docs/quick-start-guide/`

### CODE REVIEW

File: Orbital-Docs/_pages/docs/quick-start-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+layout: single
+title: Quick-Start Guide
+permalink: /docs/quick-start-guide
+sidebar:
+  nav: 'docs'
+---
+
+This quick start guide will show you how to make a simple petstore which returns a pet by name. If no pets are found, it will return a custom message.
+
+1. Follow the installation instructions to start up the designer and server
```
Comment (mariareyesfoci) [inline Orbital-Docs/_pages/docs/quick-start-guide/index.md]: is it possible to add the link to the installation page?

Comment (alexyorke) [reply Orbital-Docs/_pages/docs/quick-start-guide/index.md]: Done!

File: Orbital-Docs/_pages/docs/quick-start-guide/installing.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+`git clone https://github.com/FociSolutions/Orbital.git`
+
+`cd Orbital`
+
+`docker-compose build`
+
+`docker-compose up -d`
+
+The mock definition designer will be running on `http://localhost:4200`, and the server on `http://localhost:5001`.
```
Comment (mariareyesfoci) [inline Orbital-Docs/_pages/docs/quick-start-guide/installing.md]: we need to either change the server url to `http://localhost:5000` or change the previous line 17 to `docker run -p 5001:80 focisolutions/orbitalmock`. Although, can we verify which is the correct one? I remember having issue with running the docker image in port 5000.

**Review by mariareyesfoci — CHANGES_REQUESTED** [review]
other than the last comment I posted. I think this pr is ready to be merged.

File: Orbital-Docs/_pages/docs/quick-start-guide/installing.md (hunk 2 above)
Comment (alexyorke) [reply Orbital-Docs/_pages/docs/quick-start-guide/installing.md]: Thank you! Will change.

Comment (alexyorke) [reply Orbital-Docs/_pages/docs/quick-start-guide/installing.md]: Changed. I didn't have any issues running on port 5000 (I've always used the `README` instructions when running the containers locally.)

**Review by mariareyesfoci — APPROVED** [review]

**Review by JohnnyFoci — APPROVED** [review]

**Pull request merged**